FITist_project/user_profiles/urls.py

```diff
@@ -21,4 +22,7 @@
     url(r'^(?P<username>\w+)?/?$',
         get_profile,
         name='user_profile'),
+    url(r'^(?P<username>\w+)/workout_of_the_day$',
```

Comment:
I made it required in this one for access to their workout. 

The way the other route is set up, they would be redirected to login if there was no user or the logged in user's profile if they don't pass a username in. So, if one was logged in instead of going to the main page, they would be redirected to their profile. (I think I have that correct but I need to double check). I believe I could make the username required and it would still work (if I'm not mistaken, it has been a couple weeks since I worked on that code).